modules/sdp-interop/index.js

```diff
@@ -0,0 +1,16 @@
+/* Copyright @ 2015 - Present, 8x8 Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from './interop.js';
```

Comment:
Moves sdp-interop lib into lib-jitsi-meet for ease of development only for the duration of the project. Why do we even have them separated :P ? is anyone using it outside?